library/src/meta/document.rs

```diff
@@ -1,6 +1,24 @@
-use crate::layout::{LayoutRoot, PageElem};
+=use crate::layout::{LayoutRoot, PageElem};
```

Comment:
```suggestion
use crate::layout::{LayoutRoot, PageElem};
```

Seems like this typo causes the CI to fail.